pkgs/build-support/rust/import-cargo-lock.nix

```diff
@@ -10,6 +10,8 @@
   # Allow `builtins.fetchGit` to be used to not require hashes for git dependencies
 , allowBuiltinFetchGit ? false
 
+, fetchSubmodules ? false
```

Comment:
Shouldn't we always fetch submodules?

Reply:
That's a good question. To match cargo behavior, yes.

Reply:
This would potentially break all existing outputHashes

Reply:
iirc this is `fetchgit`'s default, I don't think that would change any hashes, though I did make a mistake when adding `allowBuiltinFetchGit`, but that can probably go in a different PR

Reply:
No, default for fetchgit is fetchSubmodules = false` :/

Reply:
Ohhh this might be an issue with the convert-to-import-cargo-lock script. I'll look into it

Reply:
This is fixed and all hashes regenerated